### DISCUSSION TITLE
Spinner color in mini cart of woocommerce

### DISCUSSION THREAD

**Discussion by klausvennik** [bootscore/bootscore#897] [Q&A] [answered]
Hi i cant find the way to change the color of the spinner of mini cart in woocommerce using bootscore.  
![Captura de pantalla 2024-12-14 094048](https://github.com/user-attachments/assets/89630b83-ef33-4200-bb50-86913350a557).  https://shop.nuevocliente.com/

**Comment by crftwrk** [comment, marked as answer]
You have just roughly overridden your primary, danger etc. colors but the origin blue/red still exists. For example the cart count badge:

```css
.bg-danger {
  background-color: #355F2E !important;
}
```

This is the wrong way. Define theme colors in child-theme's `_bootscore-variables.scss`:

```scss
$primary: #355F2E;
```

All blue is now green. Check https://bootscore.me/documentation/theme/css-scss-compiler/ how to change Bootstrap colors correctly.

**Reply by klausvennik** [reply]
Thanks the variables.scss works perfect
